mqtt/mqtt-broker/src/persist.rs

```diff
@@ -162,15 +162,15 @@ impl FileFormat for ConsolidatedStateFormat {
         fail_point!("bincodeformat.load.deserialize_from", |_| {
             Err(Error::from(ErrorKind::Persist(ErrorReason::Deserialize)))
         });
-        let state = bincode::deserialize_from(decoder)
+        let state: ConsolidatedState = bincode::deserialize_from(decoder)
             .context(ErrorKind::Persist(ErrorReason::Deserialize))?;
 
-        let state = Resolver::resolve_state(state);
+        let state: BrokerState = state.into();
```

Comment:
I think it's not necessary to specify type here.

Reply:
yea, but I think it makes it more readable